src/count/package.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Same as the bower.json, this can be removed as long as all dependencies go into the package.json